soc/arm/atmel_sam0/samd21/soc.h

```diff
@@ -63,6 +64,14 @@
 #error Unsupported GCLK1 clock source.
 #endif
 
+/** GCLK2 Frequency */
```

Comment:
The comment relates to `SOC_ATMEL_SAM0_GCLK2_FREQ_HZ` define, should be placed above it.